### PR TITLE
Fix relative date regression

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard10**

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/utils/formatDate.ts [hunk 1]
```diff
@@ -165,19 +161,53 @@ function getFormattedDate(date: Date, style: formatStyle) {
...
+/**
+ * Returns one of `vandaag` | `gisteren` | `eergisteren`.
+ * If given date is more than 2 days ago it will return `undefined`.
+ */
+export function formatRelativeDate(input: DateInput) {
+  const date = getDate(input);
+
+  if (typeof window === 'undefined') {
+    throw new Error('formatRelativeDate cannot be called server-side');
+  }
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/utils/formatDate.ts:179]:
```suggestion
  assert (window, 'formatRelativeDate cannot be called server-side') 
```

I think Next might have a function or variable to tell you if you're server-side, so maybe that's more elegant than checking window

File: packages/app/src/utils/formatDate.ts [hunk 2]
```diff
@@ -165,19 +161,53 @@ function getFormattedDate(date: Date, style: formatStyle) {
...
+    ? siteText.utils.date_today
+    : isYesterday(date)
+    ? siteText.utils.date_yesterday
+    : isDayBeforeYesterday(date)
+    ? siteText.utils.date_day_before_yesterday
+    : undefined;
+}
+
+function isDayBeforeYesterday(date: Date): boolean {
+  return isSameDay(date, subDays(Date.now(), 2));
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/utils/formatDate.ts:191]: I was going to complain about the name subDays but it's not in our hands :)

File: packages/app/src/utils/formatDate.ts [hunk 3]
```diff
@@ -165,19 +161,53 @@ function getFormattedDate(date: Date, style: formatStyle) {
...
+  return isToday(date)
+    ? siteText.utils.date_today
+    : isYesterday(date)
+    ? siteText.utils.date_yesterday
+    : isDayBeforeYesterday(date)
+    ? siteText.utils.date_day_before_yesterday
+    : undefined;
+}
+
+function isDayBeforeYesterday(date: Date): boolean {
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/utils/formatDate.ts:190]: nit; I think you can remove the return type annotation

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/utils/formatDate.ts (hunk 1 above)
Comment (VWSCoronaDashboard10) [reply packages/app/src/utils/formatDate.ts:179]: nope: https://github.com/vercel/next.js/pull/7651

**Pull request merged**